<launch>

	<include file="$(find cmvision)/cmvision2.launch" />
	<node name="target_srv" pkg="modules" type="target_srv" />
	<node name="recon_srv" pkg="modules" type="recon_srv" />

</launch>
